### ISSUE TITLE
Entity eplus.scroll rendering crash

### ISSUE THREAD

**Issue by walkerjonny** [Epoxide-MC/Enchanting-Plus#17]
I got this crash while shooting down a creeper with my bow.
Any idea why this happens?
I'm playing with the latest Enchanting Plus beta and the latest bookshelf API from curse.
http://pastebin.com/WprvyCYH

**Comment by walkerjonny** [comment]
Crash repeats now on world loading. That seems to be a critical bug then.
http://pastebin.com/qAM1Ewj8

**Comment by darkhax** [comment]
Thank you for reporting this issue. I am not 100% sure what is causing it, however I have made some changes that should fix the issue, if it is what I think it is. Could you please try loading your world using [this]() version? Also, if your world file isn't too big, could you back it up and upload it here? I want to do some further tests on that specific Item that dropped from the Creeper.

**Comment by walkerjonny** [comment]
Not exactly the same crash as I changed some settings in the eplus config file (turned off Quest mode and Scroll spawn rate to 0.0 and security to false and max enchantment amount from 5 to 1024) and then it worked. It was a Heat scroll that caused the crash.
However I reverted all settings back and got a new crash just some seconds later after hitting another monster.
http://pastebin.com/JwyjWG9z
Here is the link for the zipped world:
http://en.file-upload.net/download-11130156/TEST2.zip.html
(~ 50 MB)
Oh, I'm playing around with very OP armor and weapons (very high looting level, many enchantments). But that didn't cause any issues as I changed the eplus.cfg as described above.
